packages/template/vite-typescript/tmpl/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "ES6",
+    "allowJs": true,
+    "module": "commonjs",
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "noImplicitAny": true,
+    "sourceMap": true,
+    "baseUrl": ".",
+    "outDir": "dist",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "paths": {
+      "*": ["node_modules/*"]
+    }
```

Comment:
This configuration is rare and seems to cause other problems, such as: https://github.com/cypress-io/cypress/issues/16188

Can you describe the need for this configuration?

Reply:
I checked and this was added in 19 years. It's been 4 years, so maybe some issues were addressed by doing this before, but the point in time may now be supported. Are you comfortable testing it? 🙇‍♂️

> I noticed that this configuration was added by you, maybe you can add some information. /cc @codebytere

Reply:
Aha! indeed outdated, have made some changes. 😂